tests/test_cli.py

```diff
@@ -294,20 +408,41 @@ def test_trials_command(objective: Callable[[Trial], float]) -> None:
             storage_url,
             "--study-name",
             study_name,
-            "--format",
-            "json",
         ]
 
+        if output_format is not None:
+            command += ["--format", output_format]
+
         output = str(subprocess.check_output(command).decode().strip())
-        trials = json.loads(output)
+        trials = _parse_output(output, output_format or "table")
 
         assert len(trials) == n_trials
 
+        df = study.trials_dataframe(attrs, multi_index=True)
```

Comment:
[Note] It's a good idea to use `study.trials_dataframe` to generate expected values 👍